content/blog/2016/2016-09-30-jenkins-world-2016-wrap-up-community.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Community"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:http://cloudbees.com[CloudBees].
+
+image:/images/post-images/2016-jenkins-world-wrap/jenkins-world-banner-pic.jpg[Jenkins World 2016,height=200,role=right]
+
+This year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] conference
+was a huge milestone for the Jenkins project - the first global event for the Jenkins community.
+It brought users and contributors together to exchange ideas on the current state
+of the project, celebrate accomplishments of the past year, and look ahead at all the exiting enhancements
+coming down the pipe(line).
+
+=== Contributor Summit
+To kick off Jenkins World, we had a full day "Contributor Summit".
+Jenkins is a distributed project with contributors from all over the globe.
+Conferences like this are perfect time to get contributors together face-to-face,
+to talk through current issues and upcoming plans for the the project.
+Some key topic discussed during this summit were:
+
+* *Infrastructure* - In the past year, the Jenkins project has moved to a new website and has entered a
```

Comment:
"new website" strikes me as confusing, perhaps rephrase as 'new domain name, a statically generated website and has entered into a partnership with Microsoft to host infrastructure on Azure"